package.json

```diff
@@ -20,6 +20,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.15.5",
+    "@tailwindcss/line-clamp": "^0.2.1",
```

Comment:
I'm quite wary of adding un-maintained or poorly maintained libraries in codebase. It's an official plugin, so fine for now, but be careful in adding Tailwind plugins in codebase going forward.